tcms/rpc/tests/test_testexecution.py

```diff
@@ -622,6 +624,31 @@ def test_update_with_no_perm(self):
                 self.execution_1.pk, {"stop_date": timezone.now()}
             )
 
+    def test_create_actual_duration(self):
+        test_execution = TestExecution()
+
+        test_execution.start_date = datetime.strptime("2020-05-05 09:00:00", "%Y-%m-%d %H:%M:%S")
+        test_execution.stop_date = datetime.strptime("2020-05-06 09:00:00", "%Y-%m-%d %H:%M:%S")
```

Comment:
You don't need to parse the dates from string, you can work directly with datetime objects in Python.